DataConnectors/alcide_kaudit.json

```diff
@@ -0,0 +1,97 @@
+  "graphQueries": [
+      {
+          "metricName": "Anomalies and Incidents - All Data",
+          "legend": "alcide_kaudit_detections_1_CL",
+          "baseQuery": "alcide_kaudit_detections_1_CL"
+      }
+  ],
+  "sampleQueries": [
+      {
+          "description" : "All detections (anomalies and incidents) entries",
+          "query": "\nalcide_kaudit_detections_1_CL\n| sort by TimeGenerated\n"
+      },
+      {
+          "description" : "All audit activity for a Secret resource type, summarized count by resource namespace",
+          "query": "\nalcide_kaudit_activity_1_CL\n| where resource_type_s  == \"secrets\"\n| summarize count() by resource_namespace_s"
+      },
+      {
+          "description" : "Audit activity, summarized by principal, Type and Caller IP",
+          "query": "\nalcide_kaudit_selections_details_1_CL\n| summarize count() by principal_s, Type, caller_ip_s"
+      }
+  ],
+  "dataTypes": [
+      {
+          "name": "alcide_kaudit_activity_1_CL",
+          "lastDataReceivedQuery": "alcide_kaudit_activity_1_CL\n | summarize Time = max(TimeGenerated)\n | where isnotempty(Time)"
+      },
+      {
+          "name": "alcide_kaudit_detections_1_CL",
+          "lastDataReceivedQuery": "alcide_kaudit_detections_1_CL\n | summarize Time = max(TimeGenerated)\n | where isnotempty(Time)"
+      },
+      {
+          "name": "alcide_kaudit_selections_count_1_CL",
+          "lastDataReceivedQuery": "alcide_kaudit_selections_count_1_CL\n | summarize Time = max(TimeGenerated)\n | where isnotempty(Time)"
+      },
+      {
+          "name": "alcide_kaudit_selections_details_1_CL",
+          "lastDataReceivedQuery": "alcide_kaudit_selections_details_1_CL\n | summarize Time = max(TimeGenerated)\n | where isnotempty(Time)"
+      }
+  ],
+  "connectivityCriterias": [
+      {
+          "type": "IsConnectedQuery",
+          "value": [
+              "alcide_kaudit_activity_1_CL\n | summarize LastLogReceived = max(TimeGenerated)\n | project IsConnected = LastLogReceived > ago(30d)"
+          ]
+      }
+  ],
+  "availability": {
+      "status": 1
+  },
+  "permissions": {
+      "resourceProvider": [
+          {
+              "provider": "Microsoft.OperationalInsights/workspaces",
+              "permissionsDisplayText": "read and write permissions are required.",
+              "providerDisplayName": "Workspace",
+              "scope": "Workspace",
+              "requiredPermissions": {
+                  "write": true,
+                  "read": true,
+                  "delete": true
+              }
+          }
+      ]
+  },
+  "instructionSteps": [
+      {
+          "title": "",
+          "description": "Follow the step-by-step instructions provided in the [Alcide kAudit Installation Guide](https://get.alcide.io/hubfs/Alcide%20kAudit%20Installation%20Guide.pdf)",
```

Comment:
I went through the onboarding guidance and have couple of comments:
1. I did not find details on how to onboard and send logs / data from Alcide to Azure Sentinel. I just saw a reference to that and no entry in the table of contents or details on how can one configure to send data to Azure Sentinel. Can you please add this step to the onboarding guidance?
2. Please add a summary of how they can onboard to send logs to Azure Sentinel / mainly so that customers can quickly land to the Azure Sentinel onboarding part in the onboarding guide. The onboarding guide is huge and covers lots of options, hence summarizing would help here. 
3. Please update references of Sentinel to have Azure Sentinel.